### PR TITLE
My pull request

### PR DESCRIPTION
**Pull request by nyrkln**
## What type of PR is this? (check all applicable)

- [X ] 🚀 Added Name
- [ X] ✨ Feature
- [ X] 🌟 stared the repo
- [ ] 🐛 Grammatical Error
- [ ] 📝 Documentation Update
- [ ] 🚩 Other

## Description

Created python threading related question. 

## Add Link of GitHub Profile

https://github.com/nyrkln

### CODE REVIEW

File: Languages/Python/answers.json [hunk 1]
```diff
@@ -9,8 +9,8 @@
       "details": "Here you would speak on more information for the incorrect answer you have referenced, for more information look at the .json in the android folder."
     },
     {
-      "answer": "It is an example of how you should right the answer, keep it short.",
-      "details": "Here you would speak on more information for the incorrect answer you have referenced, for more information look at the .json in the android folder."
+      "answer": "No",
```
Comment (AndroidLion48) [inline Languages/Python/answers.json:12]: @nyrkln No is not an answer. Please input a short answer int "shortAns":

Please review Languages/Android/answers.json

File: Languages/Python/questions.json [hunk 2]
```diff
@@ -20,12 +20,12 @@
     },
     {
       "id": 2,
-      "question": "What is this example question?",
-      "details": "Expanded information on the correct answer to the question goes here",
+      "question": "Is python threading asynchronous?",
+      "details": "Can python threading module call for multiple operations at the same time, as allowed by the operating system",
```
Comment (AndroidLion48) [inline Languages/Python/questions.json:24]: @nyrkln  The "details": field is used to describe the correct answer in detail. 

Please review Languages/Android/answers.json

File: Languages/Python/questions.json [hunk 3]
```diff
@@ -20,12 +20,12 @@
...
       "questionType": "multi",
```
Comment (AndroidLion48) [inline Languages/Python/questions.json:25]: @nyrkln Please change question type to "bool" to accommodate for the true or false question.

**Review by AndroidLion48 — CHANGES_REQUESTED** [review]

**Comment by Tc2r1** [comment]
@nyrkln  Hi please make marked changes :) for reference please look in the java or android folder.

**Review by Tc2r1 — APPROVED** [review]

**Pull request merged**